doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
 Depends xxx.
 
 <a name="6.3"></a>
-## 6.3 Descriptor file definition
+## 6.3 Descriptor file definition specification
 There must be a Descriptor File definition, which used by installer as input of necessary configuration.
-Mandatory and optional definition shall be defined.
+Mandatory and optional definition shall be defined.  
+All the required definition for description file is listed in this specification session, there's no restrictions on how to use it,
+there could be multiple ways to implement PDF, the implementation will be in next session
 
 <a name="6.3.1"></a>
-### 6.3.1 Hardware resource information
-The support of different workload types, each with different compute, storage, network requirements which needs kinds of hardware configuration template, we name this as server template.
-Besides it may include optional information such area name, data center name etc.
+### 6.3.1 Resource Pool information
+This table is the description of the resource pool, it contains only 2 parameters: name and type of the resource pool.
+
+Only one instance per resource pool.
 
 | Field # | type | mandatory | Instruction |
 |----|--------------------|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| area_name | String | Yes | data center area name |
-| area_center_name | String | Yes | data center name, may compliance with a naming rule.  |
-| room_name | String | Yes | rome static data, helpful in locating issue occurred. |
-| city | String | Yes |  |
-| resource_pool_name | String | Yes |  |
-| outband_network_segment | String | Yes |  |
-| server_templates | List | Yes | server template list included in the resource pool |
-| server_infos | List | Yes | server information list included in the resource pool |
+| RES_POOL_NAME | String | Yes | This is the unique name of the resource pool, could be refered by other parameters |
+| RES_POOL_TYPE | String | Yes | User defined value to identify different hardware or software configuration requirements. |
+
+<p align="center"><b>Table 6-3-1:</b> Resource Pool Information.</p>
 
-<p align="center"><b>Table 6-3-1:</b> Hardware resource description.</p>
 
 <a name="6.3.2"></a>
-### 6.3.2 Server template
-The server template describes the capability of the host like processor, memory, harddrive, raid, NIC/NIC binding, manufacturer, model etc.
-Server template would be assigned to multiple servers, i.e physical hosts.
+### 6.3.2 Global Settings
+The Global settings are provided by the user, contains data like like IP_Type, VLAN_Type, etc.
+
+Only one instance per resource pool.
```

Comment:
Same comment as above .. instance of ?

Reply:
I have updated the description, could you help to check is it clear enough?